src/partials/advantages.html

```diff
@@ -0,0 +1,54 @@
+<section class="advantages">
+  <div class="advantages__container">
+    <ul class="advantages__list">
+      <li class="advantages__item">
+        <article class="advantages__fact">
```

Comment:
Article niepotrzebny (jakbyś zawartość umieściła na oddzielnej stronie to nie wiadomo czego miałaby dotyczyć, a article musi móc funkcjonować samodzielnie)

Reply:
ok usuwam